docs/integrations.md

```diff
@@ -1,5 +1,139 @@
 # Integrations
 
+## Vulnerability Scanners
+
+![](images/integration-vulnerability-scanners.png)
+
+### Trivy Standalone
+
+The default configuration settings enable Trivy `vulnerabilityReports.scanner` in [`Standalone`][trivy-standalone]
+`trivy.mode`. Even though it doesn't require any additional setup, it's the least efficient method. Each pod created
+by a scan job has the init container that downloads the Trivy vulnerabilities database from the GitHub releases page
+and stores it in the local file system of an [emptyDir][emptyDir-volume] volume. Such volume is then shared with main
+containers that perform the actual scanning. Finally, the pod is deleted along with the emptyDir volume.
+
+![](images/design/trivy-standalone.png)
+
+The number of containers defined by a scan job equals the number of containers defined by the scanned Kubernetes
+workload, so the cache in this mode is useful only if the workload defines multiple containers.
+
+Beyond that, frequent downloads from GitHub might lead to [rate limiting][gh-rate-limiting] problem. The limits are
+imposed by GitHub on all anonymous requests originating from a given IP. To mitigate such problems you can add the
+`trivy.githubToken` key to the `starboard` secret.
+
+```
+GITHUB_TOKEN=<your token>
+
+kubectl patch secret starboard -n <starboard_namespace> \
+  --type merge \
+  -p "$(cat <<EOF
+{
+  "data": {
+    "trivy.githubToken": "$(echo -n $GITHUB_TOKEN | base64)"
+  }
+}
+EOF
+)"
+```
+
+[trivy-standalone]: https://github.com/aquasecurity/trivy#standalone
+[emptyDir-volume]: https://kubernetes.io/docs/concepts/storage/volumes/#emptydir
+[gh-rate-limiting]: https://docs.github.com/en/free-pro-team@latest/rest/overview/resources-in-the-rest-api#rate-limiting
+
+### Trivy ClientServer
+
+You can connect Starboard to an external [Trivy server][trivy-clientserver] by changing the default `trivy.mode` from
+[`Standalone`][trivy-standalone] to [`ClientServer`][trivy-clientserver] and specifying `trivy.serverURL`.
```

Comment:
Maybe worth adding something like 
```
The Trivy Server could be your own deployment, or it could be an external service. See <Trivy docs> for more information on deploying Trivy in Client-Server mode.
```

Reply:
Done